iset.mm

```diff
@@ -143284,6 +143365,18 @@ could not be used in an extensible structure (slots must be positive
   starvslid $p |- ( *r = Slot ( *r ` ndx ) /\ ( *r ` ndx ) e. NN ) $=
     ( cstv c4 df-starv 4nn ndxslid ) ABCDE $.
 
+  ${
+    ressmulr.1 $e |- S = ( R |`s A ) $.
+    ${
```

Comment:
Probably no need of nested blocks ?

Reply:
There's a decent chance we'll want to add another theorem in another nested block in the future, but :shrug: we can bring back the nesting then if we want. I'll remove the extra nesting.

Reply:
Done.